cmd/devtool/devtool.go

```diff
@@ -133,6 +132,7 @@ func main() {
 	ethSetup(acc, keystoreDir, isBroadcaster)
 	createRunScript(acc, dataDir, serviceHost, isBroadcaster)
 	if !isBroadcaster {
+		cliPort += 1
```

Comment:
How about just hardocoding the cli ports rather than incrementing it here? We can just use:
- `7935` for broadcaster
- `7936` for orchestrator (both standalone and with transcoder)
- `7937` for transcoder

Reply:
I'd thought about hardcoding initially, but it looks like this stuff is intended to be more dynamic:

https://github.com/livepeer/go-livepeer/blob/master/cmd/devtool/devtool.go#L83-L98

Not sure how widely used it is, but it means that we can't just hardcode ports.

Reply:
👍